meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -151,8 +151,8 @@ OSTREE_KERNEL_ARGS:mx6ull-generic-bsp ?= "console=tty1 console=ttymxc0,115200 ${
 OSTREE_KERNEL_ARGS:mx7d-generic-bsp ?= "console=tty1 console=ttymxc0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
 OSTREE_KERNEL_ARGS:mx7ulp-generic-bsp ?= "console=tty1 console=ttyLP0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
 
-PREFERRED_VERSION_meson ?= "0.61.3"
```

Comment:
Please explain the reason for the revert in your commit message.

Reply:
fixed